scripts/deploy_oak_functions_loader

```diff
@@ -22,7 +21,7 @@ docker push "${FUNCTIONS_DOCKER_IMAGE_NAME}:latest"
 
 # Build and push the `weather_lookup` example application built on Oak Functions.
 
-cargo -Zunstable-options build --release \
+./scripts/docker_run cargo -Zunstable-options build --release \
```

Comment:
This is an improvement, but it's still not ideal, maybe we should split the script into two, one that builds all the things (and is expected to be run inside docker), and another just to build the docker images and deploy on GCP. Anyways we can try it out separately.

Reply:
I thought about splitting it into two, but I think it is good to have a single script that does all this. On the other hand, when all the artefacts are already built, it is easier to have a script that only builds the images and deploys to GCP. Let's fix that later.